src/impl/generated_specializations_cpp/abs/KokkosBlas1_abs_eti_spec_inst_Kokkos_complex_double__LayoutLeft_OpenMP_HostSpace.cpp

```diff
@@ -1,58 +0,0 @@
-/*
-//@HEADER
```

Comment:
This is showing as deleted and not renamed. Why ? May be github is getting confused ....